src/game_context.rs

```diff
@@ -42,6 +42,7 @@ impl GameContext {
 
         let mut level_data = self.world.write_resource::<LevelData>();
         level_data.current_level = level;
+        level_data.box_spot_identical_mode = level_json["box_spot_identical_mode"].as_bool().unwrap();
```

Comment:
`level_json["box_spot_identical_mode"]` is not exist in the level file, consider adding this entry